doc/releases/release-notes-1.14.rst

```diff
@@ -333,6 +333,9 @@ Build and Infrastructure
 * Introduced ``DT_`` prefix for all labels generated for information extracted from
   device tree (with a few exceptions, like labels for LEDs and buttons, kept
   for backward compatibility with existing applications).
+* Added :ref:`west <west>` tool for multiple repository management
+* Added support for :ref:`Zephyr modules <ext-projs>`
+* Build, flash, and debug targets now require west
```

Comment:
D'oh, I'm really sorry about this, but could you put "Build system flash and debug targets now require west"?

Of course you can still build without it...

Reply:
Since "flash" and "debug" are the targets, let's emphasize them in the text:
```
* Build system ``flash`` and ``debug`` targets now require west
```

Reply:
D'oh, guess I should pay more attention to what people write instead of blindly copy the text.